src/Api/Endpoints/Projects.php

```diff
@@ -38,7 +39,7 @@ class Projects extends Endpoint implements \LORIS\Middleware\ETagCalculator
      *
      * @return boolean true if access is permitted
      */
-    function _hasAccess(\User $user) : bool
+    private function hasAccess(\User $user) : bool
```

Comment:
@xlecours will this cause problems?

Reply:
Nop, see https://github.com/aces/Loris/pull/4971